src/main/java/org/skife/jdbi/v2/InferredColumnMapperFactory.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2015 Zane Benefits
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.skife.jdbi.v2;
+
+import com.fasterxml.classmate.ResolvedType;
+import com.fasterxml.classmate.TypeResolver;
+import org.skife.jdbi.v2.tweak.ResultColumnMapper;
+
+import java.util.List;
+
+class InferredColumnMapperFactory implements ResultColumnMapperFactory
+{
+    private static final TypeResolver tr = new TypeResolver();
+    private final Class maps;
```

Comment:
Raw types are evil, unless they are particularly onerous.  Can this be `Class<?>`?